data/core/common_data.sh

```diff
@@ -61,6 +61,20 @@ initialize_package_manager_pkg() {
   PACKAGE_MANAGER_AUTOCLEAN="pkg -y autoclean"
 }
 
+initialize_package_manager_pacman() {
+  DEFAULT_PACKAGE_MANAGER="pacman"
+  PACKAGE_MANAGER_INSTALL="pacman -S"
+  PACKAGE_MANAGER_FIXBROKEN="pacman -Sy"
+  PACKAGE_MANAGER_UNINSTALL="pacman -Ry"
+  PACKAGE_MANAGER_UPDATE="pacman -Sy"
+  PACKAGE_MANAGER_UPGRADE="pacman -Syu"
+  PACKAGE_MANAGER_INSTALLPACKAGE=":"
```

Comment:
What is the command to install a low level package? For example, In `apt` the low level installation package is `dpkg`. 

* High-level: apt install -y
* low-level: dpkg -i

Reply:
If there is not at least provide an equivalent action in the high-level package manager